doc(release): Prepare release notes for next 22.04 (#2145)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md

- [Authentication] Fixed an issue in the mixed mode to improve Web SSO feature.
- [CLAPI] Removed mandatory password for LDAP users creation.
- [Configuration] Fixed bug that occurred when a massive change was performed on services right after one of them was disabled, causing all services to get the same host and template.
- [Configuration] Fixed error in installation when chrony package already installed.
- [Configuration] Fixed export of configuration when Anomaly Detection feature is configured.
- [Core] Fixed SQL queries when databases names contain dash.
- [Core] Removed deprecated DBMS option in default configuration provided by Centreon.
- [Core] Replaced an SQL statement to fix a database compatibility limitation in an update script.
- [Install] Added missing gettext dependency for Debian.
- [Install] Fixed SQL request with MySQL 8.
- [Install] Removed app_key filed from configuration database.
- [LDAP] Fixed auto-import of users.
- [Provider] Fixed a fatal PHP error on iTop rule form when group is not an array.
- [ResourceStatus] Fixed date picker causing duplicated dates.

> - [ResourceStatus] Acknowledgement not visible in Resource status and Details after disacknowledge/ or service/host come back to OK/UP status.

Reviewer: ```suggestion
- [ResourceStatus] Fixed an issue that made the acknowledgement still active in Resource Status after a disacknowledgement or when service/host returned to OK/UP status.
```

- [UI] Fixed an issue with "display last comment" option that caused high loads and caused the interface to become unresponsive.

> - [UX] Fixed an issue that affected the "Pending" status in the top counter filter: when it was selected, the filter was not applied to deprecated  pages.

Reviewer: ```suggestion
- [UX] Fixed an issue that affected the "Pending" status in the top counter filter: when it was selected, the filter was not applied to deprecated pages.
```

> - [UX] Fixed visual incoherences in some menus.

Reviewer: ```suggestion
- [UX] Fixed visual inconsistencies in some menus.
```

#### Enhancements

- Optimized queries used to display the services monitoring page.

> - Optimized the queries used to display the hosts monitoring page.

Reviewer: ```suggestion
- Optimized queries used to display the hosts monitoring page.
```

> - Optimized the queries used to display the services grid monitoring page.

Reviewer: ```suggestion
- Optimized queries used to display the services grid monitoring page.
```

- Improved performances of event logs page.

#### Security

- [CLAPI] Fixed encoding issue.

> - Fix XSS vulnerability reflected in service-monitoring widget.

Reviewer: ```suggestion
- Fixed XSS vulnerability reflected in service-monitoring widget.
```
